graphql/fragments.graphql

```diff
@@ -28,6 +34,12 @@ fragment UserWithLanguagesFragment on User {
       ...LanguageFragment
     }
   }
+  following {
+    id
+  }
+  followedBy {
+    id
+  }
 }
 
 fragment AuthorFragment on User {
```

Comment:
Still feeling quite unsure about how the fragments are evolving and where to put these. I originally opted to put them in `UserWithStatsFragment` (wasn't too sure about that either, but it sort of made sense and we're already using that for the `author` in the `PostAuthorCard`) but then realized I needed `currentUser` to get `currentUser.following` and that uses `UserWithLanguages...`. 

Is there a point where it makes sense to have a separate query? Either separate `currentUser` queries, or simply `followersQuery` for example? Or do you think we're fine doing it like this?